metaflow/plugins/kfp/kfp.py

```diff
@@ -5,14 +5,10 @@
 from .constants import DEFAULT_FLOW_CODE_URL, DEFAULT_KFP_YAML_OUTPUT_PATH, DEFAULT_DOWNLOADED_FLOW_FILENAME
 from metaflow.metaflow_config import METAFLOW_AWS_ARN, METAFLOW_AWS_S3_REGION, DATASTORE_SYSROOT_S3
 
-from typing import NamedTuple
 from collections import deque
 
-StepOutput = NamedTuple('StepOutput', [('ds_root', str), ('run_id', str)])
-
 def step_op_func(python_cmd_template, step_name: str,
                  code_url: str,
-                 ds_root: str,
                  run_id: str,
```

Comment:
let's call this parameter the `kfp_run_id`

Reply:
And whenever we use run_id, that is for the Metaflow run_id which should start with `kfp-` just like step functions starts with `sfn-`, see https://github.com/Netflix/metaflow/blob/6e63acf8ac0ca3e0bc6999c761d4e24bb3629b5d/metaflow/plugins/aws/step_functions/step_functions.py#L633-L633.

Reply:
Done. Added the `kfp-` prefix to the run ID as well.